docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -461,6 +461,13 @@ kubeadm reset
 If you wish to start over simply run `kubeadm init` or `kubeadm join` with the
 appropriate arguments.
 
+**Note**: `kubeadm reset` will not delete any etcd data. This means that if you run 
+`kubeadm init` using the same etcd endpoints, you will see state from previous 
+clusters. To wipe etcd data after reset, it is recommended you use a client like 
+`etcdctl`, such as `etcdctl del "" --prefix`. See 
+[their documentation](https://github.com/coreos/etcd/tree/master/etcdctl) for more 
+information.
+
```

Comment:
Suggested change to text:

To wipe etc data after a reset, use a client like
[etcdctl](https://github.com/coreos/etcd/tree/master/etcdctl).
Here's an example of a command that wipes etcd data:

```shell
etcdctl del ''" --prefix
```